release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2725,6 +2725,26 @@ To work around this issue, before you update to {product-title} {product-version
 
 * There is a disk discovery delay when attaching storage to workloads. (link:https://issues.redhat.com/browse/OCPBUGS-11149[*OCPBUGS-11149*])
 
+* When upgrading from {product-title} 4.12 to 4.13, the Mellanox NIC renames SR-IOV network node policies such as `ens7f0` to `ens7f0np0` after upgrading to RHEL 9 kernel. Renaming the network node policies prevents the `SriovNetworkNodePolicy` object from locating the original interface only when the `pfNames` field is used in the policy such as `ens7f0`. Consequently, virtual functions (VFs) cannot be created because the interface cannot be found.
```

Comment:
When upgrading from {product-title} 4.12 to 4.13, the Mellanox NIC gets renamed as a consequence of moving to RHEL 9 kernel. The SR-IOV network node policies should take this renaming into account. For example, if `ens7f0` is referenced from the policy, add `ens7f0np0` too before upgrading.

Reply:
Should we be more specific and say RHEL 9.2 kernel?